### PR TITLE
[skip ci]Add comments to the file data_coord.yaml

### PR DESCRIPTION
**Pull request by Bennu-Li**
Signed-off-by: Bennu <yunmei.li@zilliz.com>

### CODE REVIEW

**Comment by Bennu-Li** [comment]
kind/improvement

**Comment by yanliang567** [comment]
@sunby could u please help to review correction of the comments for the datacoord config.
/approve

File: configs/advanced/data_coord.yaml [hunk 1]
```diff
@@ -11,6 +11,6 @@
 
 datacoord:
   segment:
-    maxSize: 512 # MB
-    sealProportion: 0.75
+    maxSize: 512 # Unit: MB, Maximum size of a segment
```
Comment (sunby) [inline configs/advanced/data_coord.yaml:14]: Maximum size of a segment in MB is better

**Comment by Bennu-Li** [comment]
@sunby I have modified the comments, please review again.

**Review by yanliang567 — COMMENTED** [review]
/lgtm

**Pull request merged**